modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp

```diff
@@ -516,25 +516,34 @@ ClaySampleUserCard claySampleUserCard = new ClaySampleUserCard();
 
 <clay:row>
 	<clay:col
-		id="image-card-block"
-		md="6"
+		id="simpleHorizontalCard"
```

Comment:
@carloslancha @markocikos I don't know what your opinion is about those `id` attributes that aren't used for anything, but for #448 I removed all of those for file cards, image cards and vertical cards.

Reply:
@julien Removing them is probably fine, I only left them in because I am not 100% sure they are not used anywhere. I am only 99% sure :slightly_smiling_face:  What is definitely wrong is having the same `id`s on different elements, which was the case here.

Reply:
I agree, the problem was the duplicated `id`s, so seems like a reasonable approach.

Reply:
Don't worry about the samples too much... @john-co, @PablitoBonito and @zhangyan0701 will take over the sample widget to use it for testing purposes, so they'll most likely remove everything and create specific test cases.